packages/grafana-ui/src/components/Select/types.ts

```diff
@@ -1,7 +1,9 @@
 import { SelectableValue } from '@grafana/data';
 import React from 'react';
+import { ActionMeta as SelectActionMeta } from 'react-select';
 
 export type SelectValue<T> = T | SelectableValue<T> | T[] | Array<SelectableValue<T>>;
+export type ActionMeta<T = any> = SelectActionMeta<SelectValue<T>>;
```

Comment:
Nope, I just did a double take and it seems the generic is not necessary at all since the `react-select` component will always pass `undefined` as `option` – probably because of the weird stuff we do in the `SelectBase` component.